server/myenv/Lib/site-packages/django/conf/__init__.py

```
@@ -0,0 +1,272 @@
+
+                if setting in tuple_settings and not isinstance(
+                    setting_value, (list, tuple)
+                ):
+                    raise ImproperlyConfigured(
+                        "The %s setting must be a list or a tuple." % setting
+                    )
+                setattr(self, setting, setting_value)
+                self._explicit_settings.add(setting)
+
+        if self.is_overridden("FORMS_URLFIELD_ASSUME_HTTPS"):
+            warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)